libraries/Microsoft.Bot.Builder.Dialogs/Prompts/PromptOptions.cs

```diff
@@ -51,5 +50,14 @@ public class PromptOptions
         /// </summary>
         /// <value>Additional options for use with a prompt validator.</value>
         public object Validations { get; set; }
+
+        /// <summary>
+        /// Gets or sets any extra properties to include in the results.
+        /// </summary>
+        /// <value>
+        /// Any extra properties to include in the results.
+        /// </value>
+        public IDictionary<string, object> Properties { get; set; } = new Dictionary<string, object>();

```

Comment:
What is this for? `Properties` doesn't seem to be used anywhere in the solution.

Reply:
Removed